static-instrumenter/agent-instrumenter/build.gradle.kts

```diff
@@ -103,10 +98,22 @@ tasks {
   assemble {
     dependsOn(shadowJar, createNoInstAgent)
   }
+}
```

Comment:
Let's make sure that suites are always run as part of the PR build:
```suggestion

  check {
    dependsOn(testing.suites)
  }
}
```

Reply:
Sure!